src/mon/OSDMonitor.cc

```diff
@@ -6115,6 +6115,11 @@ bool OSDMonitor::prepare_command_impl(MonOpRequestRef op,
 						  get_last_committed() + 1));
 	return true;
       }
+      else {
+        ss << "can not create-or-move updated item name '" << name << "' weight " << weight
+           << " at location " << loc << " to crush map";
+        goto reply;
```

Comment:
no need to `goto reply`, and the "reply" label sits right at the end of this huge `if else` block.